package.json

```diff
@@ -89,6 +89,7 @@
     "@chakra-ui/react": "^1.3.3",
     "@emotion/react": "^11.1.4",
     "@emotion/styled": "^11.1.5",
+    "@types/uuid": "^8.3.0",
```

Comment:
this is a dev dependency. Given also that you are explicitly referencing uuid in your codebase, you probably want to `npm install uuid` as well.

Reply:
addressed in 270bbc6 and d1cc058